docs/guide/migration/upgrading.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebarDepth: 2
+---
+
+# Migration from Apostrophe 2
+
+While many foundational patterns from Apostrophe 2 (A2) were maintained in Apostrophe 3 (A3), there are significant breaking changes to both the database document structure and site-building APIs. This guide will summarize those and cover how to begin upgrading website and standalone module projects from A2 to A3. Importantly, we will introduce [tools for developers to automate most of the necessary migration work](#migration-process).
+
+## Breaking changes
+
+The updates described below are the majority of significant breaking changes, but are not completely comprehensive.
+
+### Code syntax, APIs, and module configuration
+
+#### Codebase structure and module naming
+
+- The modules directory moved from `/lib/modules` to `/modules` since there was little need for the additional nesting.
+- All core modules have been namespaced with `@apostrophecms`. For example, the `apostrophe-pages` module is now `@apostrophecms/page`. You can configure this in projects in `/modules/@apostrophecms/page/index.js`.
```

Comment:
"At project level, you can configure this module in..."

Otherwise it sounds like you can configure the namespacing.

Reply:
I recognize this is probably old language, but worth tweaking.

Reply:
Please include a mention that when upgrading it is best to replace all macros with fragments, and reemphasize that when you discuss the new area template tag.